tests_and_analysis/test/euphonic_test/test_brille_interpolator.py

```diff
@@ -25,15 +26,18 @@
 # Allow tests with brille marker to be collected and
 # deselected if brille isn't installed
 pytestmark = pytest.mark.brille
-try:
-    from brille import BZMeshQdc, BZNestQdc, BZTrellisQdc
 
-    from euphonic.brille import BrilleInterpolator
-except ModuleNotFoundError:
-    pass
+
+@pytest.fixture
+def ignore_openmp_warning():
```

Comment:
I think this is already a fixture: https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#pytest-mark-filterwarnings

Reply:
Howzat?